### PR TITLE
More tests for the add-on detail page

### PR DESCRIPTION
**Pull request by AlexandraMoga**
More add-on detail stuff (a few more tests and I'll be done with it, for now at least)

### CODE REVIEW

File: tests/test_addon_detail.py [hunk 1]
```diff
@@ -442,3 +442,53 @@ def test_screenshot_keyboard_navigation(selenium, base_url, variables):
     assert '1' in addon.screenshots.screenshot_counter
     # send ESC to close the screenshot viewer
     addon.screenshots.esc_to_close_screenshot_viewer()
+
+
+@pytest.mark.nondestructive
+def test_add_to_collection_card(selenium, base_url, variables):
```
Comment (AlexandraMoga) [inline tests/test_addon_detail.py:448]: In this tests I'm interested only in checking that the Add too Collection card is displayed on the detail page. Its functionality will be tested in detail when I'll be covering Collections as a whole, which will have a dedicated test suite.

File: tests/test_addon_detail.py [hunk 2]
```diff
@@ -442,3 +442,53 @@ def test_screenshot_keyboard_navigation(selenium, base_url, variables):
...
+    assert addon.add_to_collection.collections_select_field.is_displayed()
+
+
+@pytest.mark.nondestructive
+def test_release_notes(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert (
+        'Release notes for ' + addon.more_info.addon_version_number.text
```
Comment (jrbenny35) [inline tests/test_addon_detail.py:463]: Could you use an f string here? Like how you do in the ```selenium.get()``` function?

File: tests/test_addon_detail.py [hunk 3]
```diff
@@ -442,3 +442,53 @@ def test_screenshot_keyboard_navigation(selenium, base_url, variables):
...
+
+@pytest.mark.nondestructive
+def test_more_addons_by_author_card(selenium, base_url, variables):
+    extension = variables['experimental_addon']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    # verifies that the author name from the add-on summary card
+    # is also present in the add-ons by same author card
+    assert (
+        'More extensions by ' + addon.authors.text
```
Comment (jrbenny35) [inline tests/test_addon_detail.py:477]: Same thoughts about the f string.

**Review by jrbenny35 — CHANGES_REQUESTED** [review]
Looks good, just 1 small comment.

File: tests/test_addon_detail.py (hunk 2 above)
Comment (AlexandraMoga) [reply tests/test_addon_detail.py]: Done!

File: tests/test_addon_detail.py (hunk 3 above)
Comment (AlexandraMoga) [reply tests/test_addon_detail.py]: Done!

**Review by jrbenny35 — APPROVED** [review]

**Pull request merged**